### ISSUE TITLE
SelectOneMenu: Orphaned form label

### ISSUE THREAD

**Issue by timdoering** [primefaces/primefaces#9487]
### Describe the bug

Using the SelectOneMenu a label is created with the id of the component + "_label". This label is not associated with any form control, which is causing accessibility issues.

### Reproducer

Create a SelectOneMenu:
```
            <p:outputLabel for="@next" value="Label:"/>
            <p:selectOneMenu 
                             value="#{value}"
                             label="#{label}">
                <f:selectItem itemLabel="#{placeholder}..." noSelectionOption="true"/>
                <f:selectItems value="#{selectItems}" var="item" itemValue="#{item}" itemLabel="#{item.label}" />
            </p:selectOneMenu>
```

### Expected behavior

The label should be associated with a form control or removed (or not be a label element?!)

### PrimeFaces edition

None

### PrimeFaces version

12.0.2

### JSF implementation

Mojarra

### JSF version

2.1

### Java version

11.0.14

**Comment by melloware** [comment]
Related to: https://github.com/primefaces/primefaces/issues/9279

**Comment by tandraschko** [comment]
Looks like it should not a Label tag
